dump.md

```diff
@@ -106,26 +106,34 @@ As I sort links, they may not fit in any of the categories I'm currently sorting
 
 
 ## Part 4 - Open Tools
+### Generic
 - :hammer: [Zenodo](https://zenodo.org/)
-- OLS - [Github for collaboration](https://youtu.be/7Oq81K27NmU)
-- PARTHENOS - [Virtual research environment](https://parthenos.d4science.org/web/parthenos_vre)
-- ORION [Inspiring stories](https://www.orion-openscience.eu/public/2021-07/ORION_inspiring_stories_booklet_2021_0.pdf)
-- ORION [Co-creation Menu](https://www.orion-openscience.eu/public/2018-05/D3.1%20Menu%20of%20Creation%20Tools.pdf)
-- UNESCO [Data and knowledge co-creation: a citizen science perspective](https://www.un.org/sites/un2.un.org/files/uta_wehn.pdf)
-- [European citizen science initiative](https://eu-citizen.science/)
-- PARTHENOS [Citizen science and the humanities](https://training.parthenos-project.eu/sample-page/citizen-science-in-the-digital-arts-and-humanities/)
-- [Citizen Science Italia](https://www.museonaturalemaremma.it/csi/) and [its creation announcement](https://www.nnb.isprambiente.it/it/nasce-lassociazione-nazionale-citizen-science-italia-ets)
-- [Moving forward together with Open Science](https://www.rathenau.nl/en/inclusive-science/moving-forward-together-open-science)
-- [Meaningful participation in Citizen Science, a strategy](https://www.linkedin.com/pulse/meaningful-participation-citizens-research-strategy-frederike-schmitz/)
-- Cos4EOSC - [Co-creation for EOSC](https://cos4cloud-eosc.eu/)
-- OPERAS - [COESO: Collaborative Engagement on Societal Issues](https://www.operas-eu.org/projects/coeso/)
-- OPERAS - [VERA: Enabling collaborative research with and for society](https://operas-eu.org/services/vera/) and [access to the service](https://vera.operas-eu.org/en/home)
-- Conference: [Connect, collaborate, create](https://ccc.sciencesconf.org/)
-- PHDontrack - [Pre-registration](https://www.phdontrack.net/open-science/preregistration/index.html)
-- [OSFregistries](https://osf.io/registries/) for research pre-registration
-- [Humanities Commons](https://hcommons.org/) for humanities research pre-registration
-- [Octopus](https://science-octopus.org/) - micro publications
-- JOTE - [Journal of Trial and Error](https://archive.jtrialerror.com/)
+- :arrow_forward:  OLS - [Github for collaboration](https://youtu.be/7Oq81K27NmU) Video su come usare GitHub
+- :closed_lock_with_key: 	:department_store: PARTHENOS - [Virtual research environment](https://parthenos.d4science.org/web/parthenos_vre) Service platform
```

Comment:
Many descriptions are very generic, and this is not good. I've expanded the [style guide](https://github.com/MrHedmad/data-stewardship-knowledgebase/blob/main/style_guide.md) to be more detailed about this.

Could you please re-write the terse link descriptions. For instance, this description ("service platform") does not mean anything to a reader.